### PR TITLE
feat(CX-3240): add id and email verification to edit profile page

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **TYPE**

This PR solves [CX-3240]

### Description

Added Id and Email verification

| Web | mWeb | verified status | confirmation message Web | confirmation message mWeb | 
| --- | --- | --- | --- | --- |
| image | image | image | image | image | 

[CX-3240]: https://artsyproduct.atlassian.net/browse/CX-3240?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// import { captureException } from "@sentry/react-native"
```
Comment (olerichter00) [inline src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx:1]: 👀

File: src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      // if response as state from ID verification
+      const IDVerificationState = !(
+        response && IDVerificationBlockingStates.includes(response)
+      )
+
+      if (type === "Email" || IDVerificationState) {
+        setShowVerificationBanner(true)
+      }
+    } catch (error) {
+      //  captureException(error)
```
Comment (olerichter00) [inline src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx:39]: 👀

File: src/Apps/Settings/Routes/EditProfile/helpers/verifyID.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { verifyIDMutation } from "__generated__/verifyIDMutation.graphql"
+import { commitMutation, Environment, graphql } from "react-relay"
+
+export const verifyID = async (relayEnvironment: Environment) => {
```
Comment (olerichter00) [inline src/Apps/Settings/Routes/EditProfile/helpers/verifyID.ts:4]: **suggestion:** Let's use hooks for mutations ([example](https://github.com/artsy/force/blob/2201cae43e7a07108f8f6895b4a41732a46d33d6/src/Apps/MyCollection/Routes/EditArtwork/Mutations/useCreateArtwork.ts#L5)). This is the preferred practice in Force and like this we don't have to pass the relay environment 🤓

**Review by olerichter00 — APPROVED** [review]
Great work 🌟 

I've added a few comments.

File: src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx:39]: 👀

Comment (olerichter00) [reply src/Apps/Settings/Routes/EditProfile/helpers/useHandleVerification.tsx]: 👀

**Pull request merged**